packages/patternfly-4/react-catalog-view-extension/src/components/CatalogTile/examples/CatalogTile.md

```diff
@@ -126,7 +100,7 @@ Link = () => (
       ]}
       href="http://patternfly.org/v4"
       title="Patternfly-React"
-      vendor={<React.Fragment>provided by <a href="http://redhat.com">Red Hat</a></React.Fragment>}
+      vendor="provided by Red Hat"
       description={
         'This is a very long description that should be truncated after 112 characters. ' +
         '112 is the default but can be overridden if need be. You can also provide a custom truncation function ' +
```

Comment:
I would suggest setting `maxDescriptionLength` to -1 and updating the text that references the 112 default in all the examples. IMO, we should never truncate the text since we are using the visual truncation anyhow to avoid the truncated last line not using the full width of the tile.

We should deprecate the text truncation and remove it all together in the next release.